pieman.sh

```diff
@@ -157,13 +157,13 @@ run_scripts "bootstrap"
 umount_required_filesystems
 
 info "creating image"
-dd if=/dev/zero of=${IMAGE} bs=1024 seek=$[ 1024 * 1024 * 7 ] count=1
+dd if=/dev/zero of=${IMAGE} bs=1024 seek=$[ `sudo du -s ${R}/ | cut -f1` + 150 * 1024 ] count=1
```

Comment:
It's redundant to use `sudo` in this case, because Pieman must be run either as root or using `sudo`.